packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -376,31 +378,76 @@ export const components: ThemeOptions["components"] = {
   MuiCheckbox: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <CheckIcon />,
+      indeterminateIcon: <SubtractIcon />,
     },
     styleOverrides: {
       root: ({ theme }) => ({
-        borderRadius: "4px",
-        "&:hover": {
+        width: theme.spacing(4),
+        height: theme.spacing(4),
+        padding: 0,
+        borderRadius: theme.spacing(1),
+        borderWidth: "1px",
+        borderStyle: "solid",
+        borderColor: theme.palette.grey[500],
+
+        ".MuiSvgIcon-root": {
+          color: theme.palette.common.white,
+          width: theme.spacing(3),
```

Comment:
```suggestion
          width: `${theme.typography.ui.lineHeight}em`,
```

Similar to above, this will give us the same visual result, but tie the sizing to the label copy.

Reply:
@jordankoschei-okta looks like you made these updates to Radio but not here. Intentional?

Reply:
Ignore my suggestion here; however, we will want something relative...

`${theme.typography.ui.lineHeight}em` above sets the box relative to the label copy. What proportion do we want for the check to the box? 

`0.75em` would be 12px, which will match `theme.spacing(3)`. Then if the box grows, the check will also increase in size.